src/commands/begin.rs

```diff
@@ -68,8 +69,21 @@ const ENV_EX_TEMPLATE: &str = r#"# jrny environment EXAMPLE FILE
 url = "postgresql://user:password@host:port/dbname"
 "#;
 
+/// Whether or not the given path corresponds to a directory that contains
+/// no existing .sql files
 fn is_empty_dir(p: &Path) -> Result<bool> {
-    Ok(p.is_dir() && p.read_dir()?.next().is_none())
+    if !p.is_dir() {
+        return Ok(false);
+    }
+
+    let sql_file_count = p
+        .read_dir()?
+        .filter_map(|entry_result| entry_result.ok())
+        .filter_map(|entry| entry.path().extension().map(|ext| ext.to_owned()))
+        .filter(|ext| ext == OsStr::new("sql"))
+        .count();
```

Comment:
There are several places checking the `.sql` file count and they're inconsistently handling extending casing - need utility function & consistency.